### DISCUSSION TITLE
Add padding to series_position

### DISCUSSION THREAD

**Discussion by madknave** [Neurrone/beets-audible#41] [General]
Would it be possible to add padding to this field? I know it's not very often we have series going over 10, but when it happens (looking at you Wheel of Time) my paths for organizing are a bit off.

Not an issue or anything, just something that could be considered.

**Comment by Neurrone** [comment]
I'll take a look at this but as you said, this would be lower in the backlog, as this only affects series that are more than 10 items.

I'm not sure if Beets itself is able to do this padding, you may want to check the Beets options for configuring paths.

**Comment by SenorSmartyPants** [comment]
Check out the beets inline plugin for more complex examples of path formatting
https://beets.readthedocs.io/en/stable/plugins/inline.html

Specifically the stuff like `u'%02i' % (track)`
